node/tests/TestUtilities.ts

```diff
@@ -51,6 +51,14 @@ export function getFirstResult(
     return Object.values(res).at(0);
 }
 
+export function compareMaps(
+    map: Record<string, unknown>,
+    map2: Record<string, unknown>,
+) {
+    expect(Object.keys(map)).toEqual(Object.keys(map2));
```

Comment:
Consider recursive call for map containing maps. For example, `xread` returns `Map<Map<Map<String>>>>`
Same for python